README.md

```diff
@@ -215,6 +221,58 @@ A potential supplier submits a bid for two lots as a single document:
 }
 ```
 
+A first expression of interest was evaluated and meets the qualification criteria. A second response was received, but is not yet evaluated.
+
+
+```json
+{
+  "bids": {
+    "details": [
+      {
+        "id": "1",
+        "date": "2016-12-09T01:00:00+01:00",
+        "status": "valid",
+        "description": "Expression of interest submission detailing how the bidder meets the eligibility criteria laid out in the pre-qualification phase notice.",
+        "tenderers": [
+          {
+            "id": "MEGA",
+            "name": "Mega Consortium"
+          }
+        ],
+        "countriesOfOrigin": [
+          "MX"
+        ],
+        "documents": [
+          {
+            "id": "doc-1",
+            "documentType": "eligibilityCriteria",
+            "title": "Expression of interest - evidence of eligibility."
+          }
+        ]
+      },
+      {
+        "id": "2",
+        "date": "2016-12-10T01:00:00+01:00",
+        "status": "pending",
+        "tenderers": [
+          {
+            "id": "BETA",
+            "name": "Beta Consortium"
+          }
+        ],
+        "documents": [
+          {
+            "id": "doc-2",
+            "documentType": "eligibilityCriteria",
+            "title": "Expression of interest - evidence of eligibility."
+          }
+        ]
+      }
+    ]
+  }
+}
+```
```

Comment:
Since there's presently no difference in modelling bids and EOI responses, I think this example can be removed.

If the discriminator field proposed in https://github.com/open-contracting/ocds-extensions/issues/41#issuecomment-1916477736 is added, we can add a minimal example to demonstrate the usage of that field.

```suggestion
```

Reply:
agree that a 3 example is unnecessary, I've removed this and added the new discriminator field to the other examples, turning the one that's just about submissions into an EOI submission.